templates/manifest.j2

```diff
@@ -23,15 +23,15 @@ license = "{{ data.license }}"
 {% if data.code -%}code = "{{ data.code }}"{%- endif %}
 
 [integration]
-yunohost = '>= {{ data.yunohost_required_version or '11.2'}}'
+yunohost = '>= {{ data.yunohost_required_version or '11.2.30'}}'
+helpers_version = "2.1"
 {% if data.generator_mode == "tutorial" -%}
 # List of supported archs using the dpkg --print-architecture nomenclature (amd64/i386/armhf/arm64), for example: ["amd64", "i386']
 {% endif -%}
-architectures = "{{ data.architectures }}" # TODO : handle the "all" option (no ["all"])
+architectures = "{{ data.architectures }}"
 multi_instance = {{ "true" if data.multi_instance else "false" }}
-ldap = "{{ data.ldap }}" # TODO : fixme, use actual booleans + handle the "not_relevant" value
+ldap = "{{ data.ldap }}"
```

Comment:
zmlerp it's still an issue i think x_x But yeah it shouldnt appear in the final output as it's a TODO for the app generator ... maybe we can use jinja comments

```suggestion
ldap = "{{ data.ldap }}" {# TODO : fixme, use actual booleans + handle the "not_relevant" value #}
```